nbs/models.tft.ipynb

```diff
@@ -15,20 +15,14 @@
    "source": [
```

Comment:
<p>As a general rule, interpretable is positivly associated, shouldn't we include it? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/293/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
It will be a great method addition in the future to the TFT model.
For the moment the interpretable capabilities of TFT are unavailable. I will open an issue for its inclusion.

Reply:
<p>Great! </p>